### PR TITLE
Homepage add

### PR DESCRIPTION
**Pull request by kc-si**
Definition of done:
- added top navbar bootstrap styled

### CODE REVIEW

File: app/helpers/companies_helper.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module CompaniesHelper
+end
```
Comment (marekciupak) [inline app/helpers/companies_helper.rb:4]: I think you don't need that helper for now. I would consider removing this file for now. You can add it later when you really need it :)

File: app/views/companies/_navbar.html.erb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+         </ul>
+
+         <!--
+         <div  class="mx-auto">
+            <form class="d-flex">
+               <input class="form-control mr-sm-2" type="text" placeholder="Search" aria-label="Search">
+               <button class="btn btn-outline-secondary btn-sm" type="submit" role="button">Search</button>
+            </form>
+         </div> -->
+
```
Comment (marekciupak) [inline app/views/companies/_navbar.html.erb:31]: Do you need it?

```suggestion
```

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -11,6 +11,9 @@
   </head>
 
   <body>
+    <header class='container'>
+      <%= render 'navbar' %>
```
Comment (marekciupak) [inline app/views/layouts/application.html.erb:15]: good idea :)

**Review by marekciupak — APPROVED** [review]

File: app/views/companies/_navbar.html.erb [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+         <div class="d-grid gap-2 d-md-flex justify-content-md-end">
+            <ul class="navbar-nav">
+               <li class="nav-item">
+                  <button role="button" class="btn btn-outline-success">Login</button>
+               </li>
+            </ul>
+         </div>
+      </div>
+   </div>
+</nav>
```
Comment (marekciupak) [inline app/views/companies/_navbar.html.erb:41]: One more thing:

I think this file `app/views/companies/_navbar.html.erb` does not belong to `companies` namespace. I would try to move it to `app/views/layouts/_navbar.html.erb`

File: app/views/companies/_navbar.html.erb (hunk 2 above)
Comment (kc-si) [reply app/views/companies/_navbar.html.erb:31]: No, this is one of future possibility, but not needed now, will delete it

**Pull request merged**